test/test_torch.py

```diff
@@ -6021,6 +6021,73 @@ def test_pow(self, device):
             torch.pow(m1, 1, out=out)
             self.assertEqual(out, m1)
 
+    @dtypes(*list(product(torch.testing.get_all_dtypes(include_bool=False),
+                          torch.testing.get_all_dtypes(include_bool=False))))
+    def test_float_power(self, device, dtypes):
+        base_dtype = dtypes[0]
+        exp_dtype = dtypes[1]
+        if base_dtype.is_complex or exp_dtype.is_complex:
+            dtype = torch.complex128
+        else:
+            dtype = torch.float64
+
+        base = torch.randint(-100, 100, (30,), device=device).to(dtype=base_dtype)
```

Comment:
Use `make_tensor`:

https://github.com/pytorch/pytorch/blob/49b198c45444eb44919201e6319db38b0ce443c1/torch/testing/_internal/common_utils.py#L1432

and compare with NumPy's float_power here.

Reply:
Addressed! Thanks for advice!